.travis.yml

```diff
@@ -131,6 +131,14 @@ script:
         echo -e "Variable HOST_CLOUD not set\n"
     fi
 
+  # Check whether this build was triggered by our cron job for security-updates
+  - echo "TRAVIS_EVENT_TYPE is:" && echo $TRAVIS_EVENT_TYPE
+  - >
+    if [ "$TRAVIS_EVENT_TYPE" = 'cron' ]; then
+        echo -e "Travis built triggered via configured cron job.\n"
```

Comment:
`echo "Travis built triggered via configured cron job."` is simpler to read

Reply:
Sure, no problem.